opencensus/stats/measure_to_view_map.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018, OpenCensus Authors
```

Comment:
For the very first version, I suggest not making things to complicated with the synchronization control. You could remove the locks mechanism if that makes your life easier. And we can always improve it in future PR.